guides/common/modules/proc_configuring-a-host-for-registration.adoc

```diff
@@ -1,44 +1,35 @@
 [id="Configuring_a_Host_for_Registration_{context}"]
 = Configuring a Host for Registration
 
-{RHEL} virtual machines register to Customer Portal Subscription Management by default.
-You must update each virtual machine configuration so that they receive updates from the correct {Project} Server or {SmartProxyServer}.
+Configure your host for registration to {ProjectServer} or {SmartProxyServer}.
 
 .Prerequisites
-* Hosts must be using the following {EL} version:
-** 6.4 or later
-** 7.0 or later
-* All architectures of {EL} are supported (i386, x86_64, s390x, ppc_64).
-* Ensure that a time synchronization tool is enabled and runs on the {ProjectServer}s, any {SmartProxyServers}, and the hosts.
-** For {EL} 6:
-+
-----
-# chkconfig ntpd on; service ntpd start
-----
-** For {EL} 7:
+* The host must be using a supported OS version.
+For more information, see xref:supported-clients_{context}[].
+* Ensure that a time synchronization tool is enabled and runs on {ProjectServer} and any {SmartProxyServers}.
```

Comment:
Could it be [this thingy](https://github.com/theforeman/foreman-infra/blob/4a6c278c3e20df305003ec9963f0d4515aca9110/puppet/modules/profiles/manifests/base.pp)?

Reply:
I was expecting something like this, yes. But the project is IMHO not delivered with Foreman; it's representing infra for theforeman.org and all services.

Reply:
@Lennonka foreman-infra is to maintain our own infrastructure.

> I think this should be installed by default through foreman-installer.

We don't do this today, because there are many ways to manage time management. Taking care of that is a big task and is likely to interfere with common user deployments.

On EL the easiest way to achieve this is using chrony. It's very common to be installed & configured by default. https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/using-chrony_configuring-basic-system-settings goes into it in detail and on EL guides I would certainly link to that. Also for Orcharhino because it's basic RHEL maintenance that also applies to clones.

Reply:
So I'm keeping the prerequisite.

Reply:
Isn't that more of a requirement of installing `{ProjectServer}` or `{SmartProxyServers}` in general? https://docs.theforeman.org/nightly/Installing_Server/index-katello.html#synchronizing-the-system-clock-with-chronyd_foreman & https://docs.theforeman.org/nightly/Installing_Proxy/index-katello.html#synchronizing-the-system-clock-with-chronyd_smart-proxy have very explicit instructions already so if they followed the installation guide it should already be present. Feels redundant to list it as a prerequisite.

Reply:
Good point! Then I'm removing it :D Thank you